static_src/app.jsx

```diff
@@ -0,0 +1,66 @@
+
+import React from 'react';
+
+import Login from './components/login.jsx';
+import LoginStore from './stores/login_store.js';
+import Navbar from './components/navbar.jsx';
+
+function getState() {
+  return {isLoggedIn: LoginStore.isLoggedIn()};
+}
+
+export default class App extends React.Component {
```

Comment:
@ramirezg they're kinda syntactic sugar on top of the prototype system. Here's some articles introducing them:

http://javascriptplayground.com/blog/2014/07/introduction-to-es6-classes-tutorial/
https://hacks.mozilla.org/2015/07/es6-in-depth-classes/

I still think I prefer prototypal inheritance but most new react code uses es6 classes so I'm sticking with them for now.